docs/data_usage_guides/enum_parameters/price_type_parameters.yaml

```diff
@@ -0,0 +1,3 @@
+- price type: Farm Gate
+- price type: Retail
+- price type: Wholesale
```

Comment:
I have not been able to find WFP definitions for these

Reply:
FAO have this: https://www.fao.org/economic/the-statistics-division-ess/methodology/methodology-systems/price-statistics-and-index-numbers-of-agricultural-production-and-prices/4-concepts-on-price-data/en/

Reply:
Yes I had found that, I was hesitant to use an FAO source to dexcribe WFP's data but there doesn't seem like a good publicly facing alternative.